### PR TITLE
Document how to change passwords in MongoDB

### PR DESCRIPTION
**Pull request by rhcarvalho**
Part of https://trello.com/c/ZNMI7j81
Implemented in https://github.com/openshift/mongodb/pull/80

@bparees @mfojtik PTAL

### CODE REVIEW

**Comment by mfojtik** [comment]
LGTM

**Comment by bparees** [comment]
lgtm.

File: using_images/db_images/mongodb.adoc [hunk 1]
```diff
@@ -157,6 +157,60 @@ for the database:
...
+$ oc exec <DB_PODNAME> -- bash -c 'mongo -u $MONGODB_USER -p NewPassword $MONGODB_DATABASE --eval "db.version()"'
+----
+
+Replace *<DB_PODNAME>* with the name the running MongoDB pod.
+
+To verify the admin password:
+
+---
+$ oc exec <DB_PODNAME> -- bash -c 'mongo -u admin -p NewAdminPassword admin --eval "db.version()"'
+---
```
Comment (adellape) [inline using_images/db_images/mongodb.adoc]: Should be four `-` around this. Render's currently broken as-is.

**Comment by adellape** [comment]
@rhcarvalho One minor nit, otherwise LGTM. Thanks! Assuming this targets OSE docs inclusion in 3.0.2.

File: using_images/db_images/mongodb.adoc (hunk 1 above)
Comment (rhcarvalho) [inline using_images/db_images/mongodb.adoc]: Oh yes, my bad. Fixed.

**Comment by rhcarvalho** [comment]
@adellape problem fixed. Can we merge?

**Comment by adellape** [comment]
@rhcarvalho Yep, thanks!

**Pull request merged**